crates/typst-library/src/math/equation.rs

```diff
@@ -20,8 +20,8 @@ use crate::text::{FontFamily, FontList, FontWeight, LocalName, TextElem};
 /// A mathematical equation.
 ///
 /// Can be displayed inline with text or as a separate block. An equation
-/// becomes block-level through the presence of at least one space after the
-/// opening dollar sign and one space before the closing dollar sign.
+/// becomes block-level through the presence of at least one whitespace after
+/// the opening dollar sign and one whitespace before the closing dollar sign.
```

Comment:
```suggestion
/// becomes block-level through the presence of whitespace both after
/// the opening dollar sign and before the closing dollar sign.
```

I tend to think of "whitespace" as "any amount of whitespace characters".

A similar change can be made below: "Starting and ending the equation with whitespace ..."

Reply:
Thanks! I agree, and have commited the update in 25b0f40ec5ed2c9b31b47241370ef8a2d4cbcbb2.